plugin/trino-iceberg/pom.xml

```diff
@@ -717,6 +717,13 @@
                         <ignoredResourcePattern>mozilla/public-suffix-list.txt</ignoredResourcePattern>
                         <ignoredResourcePattern>mime.types</ignoredResourcePattern>
                     </ignoredResourcePatterns>
+                    <ignoredDependencies>
+                        <dependency>
+                            <!-- duplicate and different classes in org.jspecify:jspecify-->
```

Comment:
can we exclude org.jspecify from the dependency that pulls it?

Reply:
Exclusions don't resolve duplicate failures as far as I tried.